src/handlers/tokenLocksHandler.ts

```diff
@@ -1,109 +1,282 @@
-import { PrismaClient } from '@prisma/client';
-import { APIGatewayProxyEvent, APIGatewayProxyResult } from 'aws-lambda';
-import { extractHashOrdinal, extractPagination } from '../request-params';
-import { paginatedQuery, fromCreatedAtOrdinalCursor, toCreatedAtOrdinalCursor } from '../pagination';
-import { respond, handleError, notFoundResponse } from '../response';
+import { PrismaClient } from "@prisma/client";
+import { APIGatewayProxyEvent, APIGatewayProxyResult } from "aws-lambda";
+import { extractHashOrdinal, extractPagination } from "../request-params";
+import {
+  paginatedQuery,
+  fromCreatedAtOrdinalCursor,
+  toCreatedAtOrdinalCursor,
+} from "../pagination";
+import { handleError } from "../response";
 
 const prisma = new PrismaClient();
 
-export const handleTokenLocks = async (
+const tokenLockResponse = (transaction) => ({
+  currencyId: transaction.currencyId,
+  hash: transaction.hash,
+  amount: transaction.amount,
+  source: transaction.source_addr,
+  destination: transaction.destination_addr,
+  unlockEpoch: transaction.unlock_epoch,
+  parentHash: transaction.lock_reference_hash,
+  timestamp: transaction.created_at,
+});
+
+const tokenLockResponses = (txs) => txs.map(tokenLockResponse);
+
+const tokenUnlockResponse = (transaction) => ({
+  currencyId: transaction.currencyId,
+  hash: transaction.hash,
+  amount: transaction.amount,
+  source: transaction.source_addr,
+  timestamp: transaction.created_at,
+  lockOrdinal: transaction.lock_reference_ordinal,
```

Comment:
This isn't a field on the TokenUnlock schema. We're missing `tokenLockRef` though which is pretty important - it lets us know which TokenLock this is associated with. 

Do we have that info available in the DB? If so, set this to `tokenLockRef`.

Reply:
Well actually, we also need the snapshot ordinal that this was included in so that we know when the TokenLock was unlocked.